### ISSUE TITLE
[Analyzer]  WAD

### ISSUE THREAD

**Issue by 0ssigeno** [intelowlproject/IntelOwl#814]
We can retrieve more information about the infrastructure behind a domain using [WAD](https://github.com/CERN-CERT/WAD).

Since it is actually contacting the domain, we should add the `leak_info` flag in the configuration.

**Comment by mlodic** [comment]
Good catch but we do not usually integrate tools that actively scan a target in IntelOwl.

This is because IntelOwl is not a tool that should be used for perfroming reconnaissance of a target. There are plenty of other projects that do that better and this has never been its main goal. I have already closed other similar issues. 

However I understand that the framework completely supports these use cases and could integrate several similar tools/services. The point is that we should categorize them differently. We cannot just add them as normal analyzers.

I guess that a little customization for the "reconnaissance tools/services" can be thought and done once we will start working on the playbooks (#628). The playbooks will allow to *group* some analyzers together, to better separate use cases from one another. So I think we could keep this issue as a reminder.

But right now, considering the almost all the IntelOwl users just run "all the analyzers", I think we should avoid this.

**Comment by mlodic** [comment]
this could be implemented like a normal analyzer

**Comment by basedBaba** [comment]
Hello! I would like to work on this issue. :D

**Comment by basedBaba** [comment]
Hello! I would like to work on this issue. :D

**Comment by drosetti** [comment]
ok, i'll assign it to you

**Comment by basedBaba** [comment]
This issue can now be closed. https://github.com/intelowlproject/IntelOwl/pull/2655
